public/locales/en/translation.json

```diff
@@ -164,6 +224,19 @@
       "namespaceLabel": "Namespace",
       "nameAlreadyExists": "The name already exists",
       "cancelBtn": "Cancel"
+    },
+    "button": {
+      "label": {
+        "cancel": "Cancel",
+        "create": "Create",
+        "delete": "Delete",
+        "save": "Save"
+      }
+    },
+    "dialog": {
+      "header": {
+        "confirm": "Confirmation required"
+      }
```

Comment:
Certain translation strings are re-used in different contexts. I think it would be good to have a "global" section in this file to define them. For now, I put them under "components", although they are not really strictly restricted to these components.

Reply:
Yeah, I thought about that last week too. Maybe we find something on the root level. Maybe call it `common`? We don't have to decide it now, and could make a ticket for aaron to refactor it after this was merged.

Reply:
`common` sounds good. I'm leaving it as it is right now, but created a ticket ([DIR-601]). I'll assign it to myself for now to keep track.